.github/workflows/workflow.yaml

```diff
@@ -8,9 +8,8 @@ on:
   pull_request: # This syntax stand for all PRs events
 
 env:
-  NODE_LTS_MAINTENANCE_VERSION: "14"
```

Comment:
According to the Node.js version nomenclature, we should keep MAINTENANCE and have MAINTENANCE = 16, ACTIVE = 18 and CURRENT = 20, and once Node.js 21 is released (which means 20 becomes Active LTS and 18 Maintenance LTS), we should have MAINTENANCE = 18, ACTIVE = 20, CURRENT = 21

Reply:
I'll modify that 👍🏼